src/main/scala/xitrum/view/Scalate.scala

```diff
@@ -178,35 +226,32 @@ class ScalateEngine(
    */
   protected def createContext(
     templateUri: String, engine: STE,
-    currentAction: Action, options: Map[String, Any]
-  ): (RenderContext, StringWriter, PrintWriter) =
-  {
-    val buffer     = new StringWriter
-    val out        = new PrintWriter(buffer)
-    val context    = new DefaultRenderContext(templateUri, engine, out)
-    val attributes = context.attributes
-
-    // For bindings in engine
-    attributes.update(ACTION_BINDING_ID,  currentAction)
-    attributes.update(CONTEXT_BINDING_ID, context)
-
-    // Put action.at to context
-    currentAction.at.foreach { case (k, v) =>
-      if (k == ACTION_BINDING_ID || k == CONTEXT_BINDING_ID)
-        log.warn(
-          ACTION_BINDING_ID + " and " + CONTEXT_BINDING_ID +
-          " are reserved key names for action's \"at\""
-        )
-      else
-        attributes.update(k, v)
+    currentAction: Action, options: Map[String, Any]): ScalateRenderContext =
+    {
```

Comment:
You are using 2 levels of indent (4 spaces). Please change it back to 1 level of indent (2 spaces).